src/assets/stylesheets/main/integrations/_mermaid.scss

```diff
@@ -30,7 +30,7 @@
 
   // Colors
   --md-mermaid-edge-color:             var(--md-code-fg-color);
-  --md-mermaid-node-bg-color:          var(--md-accent-fg-color--transparent);
```

Comment:
Okay, so this is actually a pretty fundamental change:

__Before__:

<img width="683" alt="Bildschirmfoto 2023-02-07 um 16 44 23" src="https://user-images.githubusercontent.com/932156/217292604-287f201a-7ba0-4c72-9a44-c801ce32c1b5.png">

__Now__:

<img width="685" alt="Bildschirmfoto 2023-02-07 um 16 44 54" src="https://user-images.githubusercontent.com/932156/217292618-4a068138-0c83-48f2-a12b-5520502c4a9f.png">

I'm afraid I don't like the changes, and expect many users to complain since they're quite fundamental. Could you please check if there's another solution? Unfortunately, this solution can't be merged.